app/components/containers/containersController.js

```diff
@@ -231,6 +231,36 @@ angular.module('containers', [])
     batch($scope.containers, Container.remove, "Removed");
   };
 
+  $scope.confirmRemoveAction = function () {
+    var isOneContainerRunning = false;
+    angular.forEach($scope.containers, function (c) {
+      if (c.Checked && c.State === 'running') {
+        isOneContainerRunning = true;
+        return;
+      }
+    });
+    if (isOneContainerRunning) {
+      ModalService.confirm({
```

Comment:
You can actually use the `ModalService.confirmDeletion` method instead of `ModalService.confirm` here.